README.md

```diff
@@ -3,7 +3,25 @@
 [![Lint Status](https://github.com/DNXLabs/terraform-aws-account/workflows/Lint/badge.svg)](https://github.com/DNXLabs/terraform-aws-account/actions)
 [![LICENSE](https://img.shields.io/github/license/DNXLabs/terraform-aws-account)](https://github.com/DNXLabs/terraform-aws-account/blob/master/LICENSE)
 
-This terraform module creates a simple AWS account.
+Terraform-aws-account is an AWS Modules that creates a simple AWS account.
+Deploy this module to your _master_ account.
+
+You will need an AWS Organization created in the master account. See `DNXLabs\terraform-aws-organization' (https://github.com/DNXLabs/terraform-aws-organization)
+
+This module requires:
```

Comment:
There is no new to have requirements here because the auto-documentation pipeline already adds this to the README.
![image](https://user-images.githubusercontent.com/18387694/90214983-a9354c80-ddd0-11ea-9696-6a3e0e195192.png)